docs/running_psalm/issues/AmbiguousConstantInheritance.md

```diff
@@ -0,0 +1,42 @@
+# AmbiguousConstantInheritance
+
+Emitted when a constant is inherited from multiple sources.
+
+```php
+<?php
+
+interface Foo
+{
+    /** @var non-empty-string */
+    public const CONSTANT='foo';
+}
+
+interface Bar
+{
+    /**
+     * @psalm-suppress OverriddenInterfaceConstant
```

Comment:
Is there a way to tell the documentation test that this should use PHP 8.1?

Reply:
See https://github.com/vimeo/psalm/blob/832fc35d8da6e5bb60f059ebf5cb681b4ec2dba5/tests/DocumentationTest.php#L312-L318